scripts/sanitycheck

```diff
@@ -1134,7 +1134,9 @@ class SanityConfigParser:
                 return vs
 
         elif typestr.startswith("set"):
-            vs = v.split()
+            vs = v.split(",")
+            if vs[0] == v:
+                vs = v.split()
```

Comment:
hmm this may not work the way you expect in all cases.
For example, if a test has "memory protection" and no other tags, it will get split as "memory" "protection"
we might want to just enforce that these values are comma separated.

Reply:
yeah, you are right, and enforcing commas would make things more consistent